.travis.yml

```diff
@@ -81,6 +84,9 @@ script:
 notifications:
   email: false
 
+after_success:
+  - bash .misc/deploy.pypi.sh
```

Comment:
for this to work, some travis secrets need to be added to the .travis.yml, and also we only want this to occur on master, not on PRs.